src/admin/content/components/ContentFilters/ContentFilters.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Button, Flex, Form, FormGroup, Spacer, TextInput } from '@viaa/avo2-components';
+
+import { CheckboxDropdownModal, DateRangeDropdown } from '../../../../shared/components';
+
+import { ContentFilterFormState, ContentTypesResponse } from '../../content.types';
+import './ContentFilters.scss';
+
+interface ContentFiltersProps {
+	contentTypes: ContentTypesResponse[];
+	formState: ContentFilterFormState;
+	hasFilters: boolean;
+	onClearFilters: () => void;
+	onFilterChange: <K extends keyof ContentFilterFormState>(
+		key: K,
+		value: ContentFilterFormState[K]
+	) => void;
+	onQueryChange: (query: string) => void;
+	query: string;
+}
+
+const ContentFilters: FunctionComponent<ContentFiltersProps> = ({
+	contentTypes,
+	formState,
+	hasFilters,
+	onClearFilters,
+	onFilterChange,
+	onQueryChange,
+	query,
+}) => {
+	// Hooks
+	const [t] = useTranslation();
+
+	// Computed
+	const contentTypeOptions = contentTypes.map(({ value }) => ({
+		checked: formState.contentType.includes(value),
+		id: value,
+		label: value,
+	}));
+
+	// Render
+	return (
+		<Spacer className="c-content-filters" margin="bottom-small">
+			<Spacer margin="bottom">
+				<Form type="inline">
+					<FormGroup className="c-content-filters__search" inlineMode="grow">
+						<TextInput
+							placeholder={t('Zoek op auteur, titel')}
+							icon="search"
+							onChange={onQueryChange}
+							value={query}
+						/>
```

Comment:
capture enter and also run onQueryChange

See existing implementation: https://github.com/viaacode/avo2-client/blob/release/v0.25.0/src/collection/components/modals/CutFragmentModal.tsx#L160-L164

Reply:
Added